test/worldwide/address_test.rb

```diff
@@ -94,20 +94,23 @@ class AddressTest < ActiveSupport::TestCase
       cl_address = Address.new(street_number: "123", country_code: "CL")
       br_address = Address.new(street_number: "123", country_code: "BR")
 
-      assert_equal " 123", cl_address.concatenate_address1
-      assert_equal " 123", br_address.concatenate_address1
+      assert_equal "⁠123", cl_address.concatenate_address1
+      assert_equal "⁠123", br_address.concatenate_address1
     end
 
     test "concatenate_address1 returns street name concatenated with street number separated by delimiter" do
-      address = Address.new(street_name: "Main Street", street_number: "123", country_code: "CL")
+      Worldwide::Region.any_instance.stubs(:combined_address_format).returns(
```

Comment:
All regions with address1 concatenation rules define a decorator. I'm using stubbing to define one without a decorator.